### PR TITLE
[IGA] Support Nitsche Condition Test

### PR DESCRIPTION
**Pull request by rickyaristio**
**Description**
This PR adds the single-patch tests for the support Nitsche conditions. This is the part of #8858 (support condition tests), which has a dependency on #8866 (support Nitsche condition).

### CODE REVIEW

**Comment by tteschemacher** [comment]
The tests look fine. Let's wait until the other branches are merged!

File: applications/IgaApplication/tests/test_IgaApplication.py [hunk 1]
```diff
@@ -60,6 +65,16 @@ def AssembleTestSuites():
...
+    if has_linear_solvers_application:
+        from KratosMultiphysics import LinearSolversApplication
+        if LinearSolversApplication.HasFEAST():
+            smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([
+                # Weak support Nitsche test
+                SinglePatchRefinedSupportNitscheTest
+                ]))
+        else:
+            print("FEAST not available in LinearSolversApplication")
+
```
Comment (tteschemacher) [inline applications/IgaApplication/tests/test_IgaApplication.py:77]: 👍

**Review by tteschemacher — APPROVED** [review]
Awesome! Thanks and well done!

File: applications/IgaApplication/tests/test_IgaApplication.py (hunk 1 above)
Comment (rickyaristio) [reply applications/IgaApplication/tests/test_IgaApplication.py:77]: Yeah, we need this as FEAST doesn't work on Windows OS. The next step is to figure out how to run the Nitsche condition on Windows, perhaps try/use another solver. I'll save it for later.

**Pull request merged**